documents/operating.md

```diff
@@ -106,6 +106,10 @@ Corporate Overlords may also vote to remove a peer from the organization if any
 
 The removal of a corporate overlord can be nominated during a plotting session and the vote must pass as defined in Section 5.
 
+### Revenge of the Corporate Overlords
+
+A former Corporate Overlord who previously left BIFFUD through one of the *voluntary* methods described under "**Removing Corporate Overlords**" (as opposed to removal via a vote of Corporate Overlords, or removal via breach of other rules in this Operating Agreement), may return to the company anew only via the full ritual described in "**Adding Corporate Overlords**."
```

Comment:
I think it's worth making explicit that they don't need to re-make their capital contribution.